bin/pyrdp-mitm.py

```diff
@@ -164,6 +164,9 @@ def main():
     parser.add_argument("--payload-powershell-file", help="PowerShell script to run automatically upon connection (as -EncodedCommand)", default=None)
     parser.add_argument("--payload-delay", help="Time to wait after a new connection before sending the payload, in milliseconds", default=None)
     parser.add_argument("--payload-duration", help="Amount of time for which input / output should be dropped, in milliseconds. This can be used to hide the payload screen.", default=None)
+    parser.add_argument("--disable-crawler", help="Disable automatic shared drive scraping", action="store_true")
+    parser.add_argument("--crawler-match-file", help="File to be used by the crawler to chose what to download when scraping the client shared drives.")
+    parser.add_argument("--crawler-ignore-file", help="File to be used by the crawler to chose what folders to avoid when scraping the client shared drives.")
```

Comment:
put default value?

Reply:
Since the configuration takes places in `bin/pyrdp-mitm.py` and the default file is in `pyrdp/mitm/crawler_config/match.txt`, I don't think moving the default value in the configuration parser is a good idea. Don't forget that the `bin/` content gets moved to the user binaries folder during installation, and we lose direct path references.

Reply:
makes sense